test/test-dashboardController.js

```diff
@@ -49,8 +49,6 @@ describe('dashboardController', function() {
         patient.isStudentStatusVerified=false;
         hasFolderCompleted(patient).should.equal(false);
         patient.isStudentStatusVerified=true;
-        patient.INE='';
```

Comment:
je trouve ce test difficile à suivre, il change bcp de champs les uns apres les autres, je ne comprends pas la stratégie du test.
On pourrait peut etre isoler les cas, genre "should return false if no doctorName", "should return false if no XYZ" etc

Si on veut etre rigoureux il faudrait tester qu'un INE vide fait qd même un dossier valide.

Reply:
bonne remarque!